crawl-ref/source/spl-data.h

```diff
@@ -3071,7 +3005,7 @@ static const struct spell_desc spelldata[] =
 
 {
     SPELL_AURA_OF_BRILLIANCE, "Aura of Brilliance",
-    spschool::charms,
+    spschool::hexes,
```

Comment:
also odd

Reply:
swapped to conjurations; monster buffs sadly don't have a good thing to put here, but it's not used.